code/linear_svm.py

```diff
@@ -0,0 +1,54 @@
+#All the libraries we need for linear SVM
```

Comment:
All python comments should have a space after the # for cleanliness & consistency